.eslintrc.json

```diff
@@ -13,10 +13,16 @@
   ],
   "parser": "@typescript-eslint/parser",
   "rules": {
+    "quotes": "off",
     "max-len": [
       2,
-      100
+      200
     ],
-    "prettier/prettier": "warn"
+    "prettier/prettier": [
+      "warn",
+      {
+        "endOfLine": "auto"
```

Comment:
ich würde stark dafür plädieren, dass wir unix line endings verwenden.
Man kann seine IDE so einstellen, dass sie die Line Endings richtig setzt (unix line endings sind die einzig richtigen).

Mit der eslint regel: "linebreak-style": ["error", "unix"]
fixed eslint das problem für dich.

übrigens wird in .gitattributes das eh wieder überschrieben.